### PR TITLE
Fixup #2658 Intermittent failures in MongoDbTest.testTailingConsumer(…

### PR DESCRIPTION
**Pull request by ppalaga**
…) and MongoDbTest.testPersistentTailingConsumer()

The tests are still failing intermittently. I have added some debug code and increased the awaitility timeout

### CODE REVIEW

**Comment by ppalaga** [comment]
cc @JiriOndrusek

File: integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java [hunk 1]
```diff
@@ -277,13 +280,19 @@ public void testOutputTypeDocument() throws Exception {
...
     private void waitAndResetTailingResults(int expectedSize, String laststring, String resultId) {
-        await().atMost(5, TimeUnit.SECONDS).until(
-                () -> RestAssured
-                        .given().contentType(ContentType.JSON)
-                        .get("/mongodb/results/" + resultId)
-                        .then()
-                        .statusCode(200)
-                        .extract().as(Map.class),
+        LOG.info("Awaiting results with size " + expectedSize + " and last string " + laststring);
+        await().atMost(30, TimeUnit.SECONDS).until(
```
Comment (zbendhiba) [inline integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java:284]: I hope this is enough for CI build. I used 20 seconds on my machine yesterday and it was still failing.

**Review by zbendhiba — APPROVED** [review]

File: integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java (hunk 1 above)
Comment (ppalaga) [reply integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java:284]: Let's see if the timeout helps, and also let's see what the debug output will be when it fails next time.

Comment (JiriOndrusek) [reply integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java:284]: @zbendhiba If it failed with 20 seconds on your machine, was it successful with some higher timeout?
(I remember, that it was able to finish on my machine in less then a few seconds and if I remember correctly,  Peter and Alex were running it ~10 times both and it was successful)

**Pull request merged**